### PR TITLE
Add `open docs` command

### PR DESCRIPTION
**Pull request by ryan-blunden**
Make it easier to open our docs without manually opening in the browser.

Closes ENG-2588

### CODE REVIEW

File: pkg/cmd/open.go [hunk 1]
```diff
@@ -80,10 +80,23 @@ var openGithubCmd = &cobra.Command{
 	},
 }
 
+var openDocsCmd = &cobra.Command{
+	Use:   "docs",
+	Short: "Open the Doppler documentation home page",
+	Args:  cobra.NoArgs,
+	Run: func(cmd *cobra.Command, args []string) {
+		err := open.Run("https://dashboard.doppler.com/workplace/docs")
```
Comment (nmanoogian) [inline pkg/cmd/open.go]: For good measure, should this be based on the `DashboardHost`?

Comment (rgmvisser) [reply pkg/cmd/open.go]: Should we just do `https://docs.doppler.com/`? In case someone is not logged in?

**Review by nmanoogian — DISMISSED** [review]
Nice! Can we squash those commits?

File: pkg/cmd/open.go [hunk 2]
```diff
@@ -80,10 +80,23 @@ var openGithubCmd = &cobra.Command{
 	},
 }
 
+var openDocsCmd = &cobra.Command{
+	Use:   "docs",
+	Short: "Open the Doppler documentation home page",
```
Comment (Piccirello) [inline pkg/cmd/open.go]: super nit: can you match the capitalization of the other `open` commands

**Review by Piccirello — APPROVED** [review]

**Pull request merged**